services/brig/test/integration/API/User/Client.hs

```diff
@@ -489,6 +489,8 @@ testRemoveClient hasPwd brig cannon = do
           newClientCookie = Just defCookieLabel
```

Comment:
I cannot mark the correct line in this file but there is a `@END` tag missing around line 446

Reply:
@fisx I see also that in this file there are spaces missing between `@SF.Channel` and `@TSFI.RESTfulAPI` in the lines before:
* "rejects saml responses with invalid issuer entity id"
* "rejects saml responses signed with the wrong private key"
* "rejects saml responses to requests not in cassandra:spar.authreq"
* "rejects saml responses already seen (and recorded in cassandra:spar.authresp)"

Additionally please change `@SF.CHANNEL` into `@SF.Channel`